include/boost/math/interpolators/detail/cardinal_cubic_b_spline_detail.hpp

```diff
@@ -170,7 +170,7 @@ cardinal_cubic_b_spline_imp<Real>::cardinal_cubic_b_spline_imp(BidiIterator f, B
     m_beta.resize(length + 2, std::numeric_limits<Real>::quiet_NaN());
 
     // Since the splines have compact support, they decay to zero very fast outside the endpoints.
-    // This is often very annoying; we'd like to evaluate the interpolant a little bit outside the
+    // This is often very annoying; we'd like to evaluate the interpolant a little outside the
```

Comment:
We could argue this one forever, but the original text makes perfect sense to me.